### ISSUE TITLE
Notes are loaded and displayed instead of plugins/tabs in the sidebar

### ISSUE THREAD

**Issue by rogor** [alangrainger/obsidian-lazy-plugins#5]
Hi, 

The bug is tough to explain with words, so I'll explain with pictures.

This was the setup before closing Obsidian - notice that the selected tab on the left sidebar is the *Recent files* plugin, set to load after 10s: ![before_closing](https://github.com/user-attachments/assets/f7b55fc8-0ed5-46b8-af27-ba074d4f4d93)

I restart Obsidian, and now the left sidebar contains a new tab, that contains the note that was open in the main window: ![new_tab_content](https://github.com/user-attachments/assets/8f9260f9-27d2-4184-9a78-76439261ec70)
And then every restart adds a new tab (all the same note). I once had the small calendar in the bottom right replaced with a note as well (though I can't reproduce it).
It only happens when the currently selected tab in the sidebar is the plugin that has its loading delayed.

Basically, it seems like the note is loaded instead of the plugin, then the plugin loads next to it. I made a video of the process here if it's not clear: [2024-08-30 23-41-40.webm](https://github.com/user-attachments/assets/28b657f8-3136-430a-8bd7-eaeca4ee3049)

On PC, it's fixable by clicking the note and hitting CTRL+W to close it, but I had the same problem on mobile as well and I had to go into the workspace-mobile.json file to fix it.

Is it fixable, or does it just mean that I shouldn't delay the loading of any plugin that displays something in the sidebars ?

Thanks

**Comment by alangrainger** [comment]
I also use the Recent Files plugin, and I can't reproduce this.

The lazy loader plugin doesn't really do anything weird or special, it just uses Obsidian's built-in functionality to disable or enable plugins.

I suggest:

1. Create a test vault and adding the Recent Files plugin and Lazy Loader plugin to confirm that this behaviour you're experiencing does _not_ occur in the test vault.
2. Keep adding plugins and your custom configurations to identify what exactly is causing this to happen.

If you can identify it, I can try and fix it. But currently I don't experience the same issue with Recent Files so we need some more info.

**Comment by alangrainger** [comment]
Can you also try with 1.0.13 and set Recent Files to **Instant** load type, and see if that resolves the problem.

**Comment by rogor** [comment]
I rolled back to 1.0.12, and I found where the problem comes from: the **daily note** plugin.  The problem occurs when I have the option **Open daily note on startup** active.

**Comment by alangrainger** [comment]
Sorry about the all-instant-loading bug, it is fixed now in 1.0.15.

If you can try that with Recent Files set to **Instant** and let me know please :)

**Comment by rogor** [comment]
It seemed to work at first, but I tried a few things and managed to recreate the problem in a new way:
Every plugin is on **short delay**, except _Recent Files_ which is on **instant**. If I move the calendar plugin to the left sidebar, and the note appeared again on the left instead of leaving a spot for the calendar, but the calendar loaded in its default spot on the right instead of where I had put it.
Video: 
[2024-09-02 23-29-59.webm](https://github.com/user-attachments/assets/72f9408d-a254-4503-ae62-7954f52efbb3)

And I restarted Obsidian again immediately after, and this time the note loaded on the right sidebar:
![Screenshot 2024-09-02 233507](https://github.com/user-attachments/assets/3cf63eaa-854d-4d7c-993c-3f361a72a0ba)

I've found two ways of fixing the problem, either:
- Disable **Open daily note on startup** in the Daily note plugin, or
- Set every plugin that displays in the sidebars to **Instant**

It seems to me that the issue doesn't actually come from your plugin, but from the way the Daily note plugin loads the note on startup.

**Comment by alangrainger** [comment]
That makes sense - it sounds like the Daily plugin is not using the correct method of targeting the main note area, and is instead targeting whatever pane is most recently active - in this case the sidebar due to a plugin loading.

I will close this issue as there sadly isn't much I can do about that, and you have at least a workaround.

Another workaround I could suggest is try using one of the other plugins which can open a Daily Note - for example the [Homepage plugin](https://obsidian.md/plugins?id=homepage) and see if that works better.
